pkg/sql/logictest/testdata/logic_test/scrub

```diff
@@ -7,24 +7,19 @@ EXPERIMENTAL SCRUB TABLE t
 statement ok
 CREATE TABLE t (
   id int PRIMARY KEY,
-  name string,
-  CONSTRAINT abc CHECK (name > 'he')
+  name STRING,
+  data INT DEFAULT 2,
+  CONSTRAINT abc CHECK (name > 'he'),
+  INDEX name_idx (name)
 )
 
 statement ok
-INSERT INTO t VALUES (1, 'hello')
+INSERT INTO t VALUES (1, 'hello'), (2, 'help'), (0, 'heeee')
 
 query TTTTTTTT
 EXPERIMENTAL SCRUB TABLE t
 -----
 
-query TTTTTTTT
-EXPERIMENTAL SCRUB TABLE t WITH OPTIONS INDEX ALL
------
```

Comment:
Why did you take this out?

The tests for this change are very subtle. Can we do something about that?

Reply:
Welp, didn't mean to take that out. Added a short comment also mentioning what these are testing. These are testing more of the options than scrub, as the single run of scrub would be doing everything (so the rest is redundant).